layouts/shortcodes/langchoose.html

```diff
@@ -1,11 +1,53 @@
+{{/* Specify `nopreview` to hide preview badges. */}}
+{{ $nopreview := false }}
+
+{{ $typescript := true }}
+{{ $javascript := true }}
+{{ $python := true }}
+{{ $go := true }}
+{{ $csharp := false }}
```

Comment:
Any reason we chose the opposite polarity than the other languages?

Reply:
For expedience, I made these opt-in for now, otherwise they'd show up in places where we don't yet have content for those languages. Subsequently, would definitely like to make the options to this shortcode more consistent. Cyrus was wondering if we could just make the tabs show up automatically based on the content below.